packages/flutter/lib/src/widgets/pop_scope.dart

```diff
@@ -10,6 +10,12 @@ import 'routes.dart';
 
 /// Manages system back gestures.
 ///
+/// A system back gesture is a swipe inward from near the edge of the screen on
```

Comment:
Ah so this widget is a no-op if used on any other platforms other than Android? Maybe should make it to the short description line: "Manages system back gestures on Android." ?

Reply:
I've rearranged this to explain it better too.